### PR TITLE
V1.7.9 patchset

### PR DESCRIPTION
**Pull request by squat**
Dev tests are passing. Carries patches from 1.7.8, including Calico 1.11.0.

cc @s-urbaniak @aaronlevy

### CODE REVIEW

File: cluster/images/hyperkube/Makefile [hunk 1]
```diff
@@ -84,6 +84,8 @@ else
 endif
 	# Download CNI
 	curl -sSL --retry 5 https://storage.googleapis.com/kubernetes-release/network-plugins/cni-${ARCH}-${CNI_RELEASE}.tar.gz | tar -xz -C ${TEMP_DIR}/cni-bin
+	curl -sSL --retry 5 -o  ${TEMP_DIR}/cni-bin/bin/calico https://github.com/projectcalico/calico-cni/releases/download/v1.11.0/calico
+	chmod +x ${TEMP_DIR}/cni-bin/bin/calico
```
Comment (s-urbaniak) [inline cluster/images/hyperkube/Makefile]: As discussed last week: we should make sure that the flannel binaries are in sync with https://github.com/coreos/tectonic-installer/blob/master/config.tf#L69.

**Review by s-urbaniak — CHANGES_REQUESTED** [review]
I have no objections modulo the flannel binaries sync. We could take this release as the precedence to:

1. sync the flannel binaries with the expected ones in installer's 1.7.9
2. In addition start shipping the `portmap` plugin

**Comment by aaronlevy** [comment]
cc @dghubble - do you know if there are recommended calico-CNI versions for v1.7.x?

**Comment by dghubble** [comment]
Calico docs still show cniVersion 0.1.0, so I don't think we'll find a recommendation there. https://docs.projectcalico.org/v2.6/getting-started/kubernetes/installation/hosted/kubernetes-datastore/calico-networking/1.7/calico.yaml.

Calico 2.6.x (corresponds to plugin v1.11.0) requires Kubernetes v1.7.x or higher so its probably a good choice. The `quay.io/calico/cni:v1.11.0` sidecar installs the v1.11.0 Calico plugin and I'm using it on Kubernetes 1.7 and 1.8 with cniVersion 0.3.0.

**Review by s-urbaniak — APPROVED** [review]

**Comment by squat** [comment]
Dev tests succeeded. Running one more test locally and then merging.

**Comment by squat** [comment]
Clusters are coming up happily. Merging.

**Pull request merged**